network.c

```diff
@@ -320,7 +320,7 @@ static char * network_get_description(struct addrinfo *res)
 				if (errno == 0) {
 					/* Windows uses numerical interface identifiers */
 					ptr = description + strnlen(description, len) + 1;
-					iio_snprintf(ptr, IF_NAMESIZE, "%u", in->sin6_scope_id);
+					iio_snprintf(ptr, IF_NAMESIZE, "%lu", in->sin6_scope_id);
```

Comment:
Unrelated?

Reply:
On MinGW, this is the output when using `%u`:
D:\a\1\s\network.c:323:74: error: format '%u' expects argument of type 'unsigned int', but argument 4 has type 'u_long' {aka 'long unsigned int'} [-Werror=format=]

Reply:
That's fine, but I'd prefer it into its own commit then. Because you're not just touching CI files but a core source file.

Reply:
> That's fine, but I'd prefer it into its own commit then. Because you're not just touching CI files but a core source file.

+1

Reply:
Comment updated.
Thanks